### PR TITLE
Clarify how to stop the bot via grammY runner

### PR DESCRIPTION
**Pull request by KnorpelSenf**
In response to https://github.com/grammyjs/runner/issues/22

### CODE REVIEW

**Comment by andrew-sol** [comment]
Looks pretty clear.

**Review by rojvv — APPROVED** [review]

File: site/docs/uk/plugins/runner.md [hunk 1]
```diff
@@ -120,9 +120,13 @@ Runner вирішить всі необхідні обмеження на льо
 ```ts
 const handle = run(bot);
 
+// Він буде викликаний, коли бот зупиниться.
 handle.task().then(() => {
   console.log("Бот завершив обробку!");
 });
+
+// Пізніше зупиніть бота за допомогою керуючого елементу runner'а.
```
Comment (niusia-ua) [inline site/docs/uk/plugins/runner.md:128]: @LWJerri @deptyped
Is `керуючий елемент` a good translation of `handle`?
A direct translation doesn't seem to fit the purpose.

Comment (KnorpelSenf) [reply site/docs/uk/plugins/runner.md:128]: Other related words would be
- grip
- knob
- holder

Also note that the variable is called `handle` and that its type is `RunnerHandle` so you could also use the English term, but I'm not sure if that fits very well here.

File: site/docs/id/plugins/runner.md [hunk 2]
```diff
@@ -117,9 +117,13 @@ Kamu juga bisa menunggu runner berhenti dengan cara menunggu promise `task`---me
 ```ts
 const handle = run(bot);
 
+// Ini akan dipanggil ketika bot berhenti.
 handle.task().then(() => {
   console.log("Bot selesai memproses!");
 });
+
+// Kemudian, hentikan bot melalui fungsi handler-nya
```
Comment (KnorpelSenf) [inline site/docs/id/plugins/runner.md:125]:
```suggestion
// Kemudian, hentikan bot melalui fungsi handler-nya.
```

File: site/docs/uk/plugins/runner.md (hunk 1 above)
Comment (LWJerri) [reply site/docs/uk/plugins/runner.md:128]: I don't know, this translation is fine with me as well.

**Review by LWJerri — APPROVED** [review]

**Review by quadratz — APPROVED** [review]

**Pull request merged**